lib/cmds/pull-request.js

```diff
@@ -418,15 +419,15 @@ PullRequest.prototype.getBranchNameFromPullNumber_ = function(number) {
     }
 }
 
-PullRequest.prototype.getPullRequestNumberFromBranch_ = function() {
+PullRequest.prototype.getPullRequestNumberFromBranch_ = function(currentBranch, prefix) {
```

Comment:
What is the purpose of adding the `currentBranch` & `prefix` params if we are not providing these arguments when calling `getPullRequestNumberFromBranch_()`? The only place I see them provided is in the test you wrote.

Reply:
Thanks for writing the test by the way!

Reply:
Possibly optional fallback for testing purposes. I think this function is more easily tested when you can control what inputs you get.

Keep up the work!

Reply:
@protoEvangelion it was just so I could write the test.  I figured I'd change as little as possible.  I did think about changing the caller too, which may ultimately be simpler (and possibly better since the test will then cover what the actual usage looks like).  I can resend with that change if you'd prefer.